reascripts/ReaSpeech/source/ASRControls.lua

```diff
@@ -10,6 +10,11 @@ ASRControls = PluginControls {
   DEFAULT_LANGUAGE = '',
   DEFAULT_MODEL_NAME = 'small',
 
+  HELP_MODEL = 'Model to use for transcription. Larger models provide better accuracy but use more resources.',
```

Comment:
would it be worth it to expand on "more resources?" like, "...but use more resources like disk space and memory."

Reply:
Good call. I might tweak the word wrap settings on these tooltips so that longer messages occupy more than one line.